apps/nextjs-website/package.json

```diff
@@ -23,6 +23,7 @@
     "eslint-config-next": "13.4.19",
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-prettier": "^5.0.0",
-    "typescript": "5.2.2"
+    "typescript": "5.2.2",
+    "synckit": "^0.8.5"
```

Comment:
what do you need this dependency for? is it really needed based on the scope of this activity?

Reply:
Commit wouldn't work without it.

Reply:
What error do you have on commit if you remove `synckit`?

Reply:
None of course. It fixed itself. It had something to do with lint so I would assume setting the configuration fixed the problem.

Already committed the change, thank you!